README.md

```diff
@@ -1,2 +1,6 @@
 # code_style
 Code styles, and tools for different languages
+
+# General conventions
+
+- Mobile Clients will send a header `bv-app-version` with the app version for each api request.
```

Comment:
+1 for `"bv-user-agent" "platform-version"`
with `platform` being either `android` or `ios` and version following semver standars